src/Result/RecordIterator.php

```diff
@@ -170,17 +170,17 @@ protected function queryMore()
      *
      * @return int
      */
-    public function count()
+    public function count(): int
     {
         return $this->queryResult->getSize();
     }
 
     /**
      * @param int $position
      */
-    public function seek($position)
+    public function seek(int $position): void
     {
-        return $this->getObjectAt($position);
+        $this->getObjectAt($position);
```

Comment:
Could be considered a compatibility break, but probably unlikely to affect anyone in practice? Open to feedback or alternatives.